lightning/src/ln/features.rs

```diff
@@ -52,6 +52,8 @@
 //!     (see [BOLT-2](https://github.com/lightning/bolts/blob/master/02-peer-protocol.md) for more information).
 //! - `Keysend` - send funds to a node without an invoice
 //!     (see the [`Keysend` feature assignment proposal](https://github.com/lightning/bolts/issues/605#issuecomment-606679798) for more information).
+//! - `AnchorsZeroFeeHtlcTx` - requires/supports that commitment transactions include anchor outputs and HTLC transactions have zero fee
```

Comment:
Can say "HTLC transactions are pre-signed with zero fee" The transactions they have fees, they're just added a posteriori